prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -469,15 +540,23 @@ class Hl7Serializer(val metadata: Metadata) {
         terser.set("SFT-6", buildDate)
 
         terser.set("/PATIENT_RESULT/PATIENT/PID-1", "1")
+        terser.set("/PATIENT_RESULT/PATIENT/PID-3-4-3", "ISO")
 
         terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/ORC-1", "RE")
+        terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/ORC-2-4", "ISO")
+        terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/ORC-3-4", "ISO")
 
         terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/OBR-1", "1")
+        terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/OBR-2-4", "ISO")
+        terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/OBR-3-4", "ISO")
 
         terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/SPECIMEN/SPM-1", "1")
+        terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/SPECIMEN/SPM-2-1-4", "ISO")
+        terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/SPECIMEN/SPM-2-2-4", "ISO")
 
         terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/OBSERVATION/OBX-1", "1")
         terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/OBSERVATION/OBX-2", "CWE")
+        terser.set("/PATIENT_RESULT/ORDER_OBSERVATION/OBSERVATION/OBX-23-7", "XX")
     }
```

Comment:
See my OBX-23-3 comment above.

Reply:
I will verify. None of the validation has thrown this error, but it's best for me to double check.

Reply:
No. OBX-23-3 is old, and only maintained for backward compatibility. It is recommended instead that we use OBX-23-10, which we are.